### PR TITLE
Feature/dsh 6 whatisit component

### PR DESCRIPTION
**Pull request by CQW-Code**

### CODE REVIEW

File: assets/templates/whatisitcomp.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    <title>Skookum DayShift 2018</title>
+
+    </head>
+
+    <body>
+      <!--What is it? component- left side of three columns
+        explains what the Dayshift week long program is-->
+      <section>
+        <img src="./assets/images/hand.png"/>
+        <h3 classs="what">What is it?</h3>
```
Comment (phatstrat) [inline assets/templates/whatisitcomp.html]: typo in class declaration "classs"(sic)

File: assets/templates/whatisitcomp.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
```
Comment (phatstrat) [inline assets/templates/whatisitcomp.html]: HTML Boilerplate unnecessary

File: assets/templates/whatisitcomp.html [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+      <!--What is it? component- left side of three columns
+        explains what the Dayshift week long program is-->
+      <section>
+        <img src="./assets/images/hand.png"/>
+        <h3 classs="what">What is it?</h3>
+        <ul>
+          <li>Immersive two week program lead by 
+            Charlotte's leading design, strategy, and development
+            agency Skookum.</li>
+          <li>Choose from our design week or our development week...
```
Comment (phatstrat) [inline assets/templates/whatisitcomp.html]: no ellipses in the li tag

File: assets/templates/whatisitcomp.html (hunk 1 above)
Comment (liamnaddell) [reply assets/templates/whatisitcomp.html]: Should this include alt text?

**Pull request merged**